orchestra/google_apps/service.py

```diff
@@ -27,26 +27,27 @@ def _create_drive_service(self, google_p12_path,
         http_auth = credentials.authorize(Http())
         return build('drive', 'v2', http=http_auth, cache_discovery=False)
 
-    def insert_file(self, title, description, parent_id, mime_type, filename):
+    def insert_file(self, title, description, parent_id, file_mime_type,
+                    google_mime_type, filename):
         """Insert new file.
 
         Args:
           service: Drive API service instance.
           title: Title of the file to insert, including the extension.
           description: Description of the file to insert.
           parent_id: Parent folder's ID.
-          mime_type: MIME type of the file to insert.
+          file_mime_type: MIME type of the file to insert.
+          google_mime_type: MIME type of the file in Google Drive.
```

Comment:
Turns out we were passing the same MIME type for both the file and the Google Drive doc type, which worked in the two existing use cases in convenience.py above (refactored appropriately) but not for our use case of uploading a CSV and having it be converted to a Google Sheet.